Google Cloud/google-folder/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  folders_yml = fileexists(var.folders_yml) ? file(var.folders_yml) : null
+  folders = local.folders_yml == null ? {} : yamldecode(local.folders_yml)
+  parent_id = lookup(local.folders.components, "parent_id", {})
+  parent = can(regex("^folders/", local.parent_id)) ? data.google_folder.self[0].name : data.google_organization.self[0].name
+  names = toset(lookup (local.folders.components, "display_name", []))
```

Comment:
The name may contain letters, digits, spaces, hyphens and underscores. Please make the ID and display name separate as in the ticket.